reference/OpenAPI.json

```diff
@@ -32561,144 +32318,10 @@
-                "default": "campaign",
-                "enum": [
-                  "campaign"
-                ]
-              }
-            }
-          },
-          "object": {
-            "type": "string",
-            "default": "earning_rule",
-            "enum": [
-              "earning_rule"
-            ],
-            "description": "The type of the object represented by JSON. Default is earning_rule."
-          },
-          "automation_id": {
-            "type": "string",
-            "description": "For internal use by Voucherify."
-          },
-          "start_date": {
-            "type": "string",
-            "description": "Start date defines when the earning rule starts to be active. Activation timestamp is presented in the ISO 8601 format. Earning rule is inactive before this date. If you don't define the start date for an earning rule, it'll inherit the campaign start date by default."
-          },
-          "expiration_date": {
-            "type": "string",
-            "description": "Expiration date defines when the earning rule expires. Expiration timestamp is presented in the ISO 8601 format. Earning rule is inactive after this date.If you don't define the expiration date for an earning rule, it'll inherit the campaign expiration date by default."
-          },
-          "validity_timeframe": {
-            "$ref": "#/components/schemas/ValidityTimeframe"
-          },
-          "validity_day_of_week": {
-            "$ref": "#/components/schemas/ValidityDayOfWeek"
-          },
-          "validity_hours": {
-            "$ref": "#/components/schemas/ValidityHours"
-          },
-          "metadata": {
-            "type": "object",
-            "description": "The metadata object stores all custom attributes assigned to the earning rule. A set of key/value pairs that you can attach to an earning rule object. It can be useful for storing additional information about the earning rule in a structured format."
-          }
-        },
-        "required": [
-          "id",
-          "created_at",
-          "loyalty",
-          "source",
-          "object",
-          "automation_id",
-          "metadata"
-        ]
-      },
       "EarningRuleEvent": {
         "title": "Earning Rule Event",
-        "type": "string",
-        "enum": [
-          "order.paid",
-          "customer.segment.entered",
-          "custom_event",
-          "customer.loyalty.tier.upgraded",
-          "customer.loyalty.tier.downgraded",
-          "customer.loyalty.tier.prolonged",
-          "customer.loyalty.tier.joined",
-          "customer.loyalty.tier.left"
-        ]
+        "example": "\"order.paid\", \"customer.segment.entered\", \"customer.loyalty.tier.upgraded\", \"customer.loyalty.tier.downgraded\", \"customer.loyalty.tier.prolonged\", \"customer.loyalty.tier.joined\", \"customer.loyalty.tier.left\"",
```

Comment:
Nawet jeśli jednak wrzucamy to do przykładu, to lepiej to zrobić w inline code tagach "`".